src/send_body.rs

```diff
@@ -64,6 +64,7 @@ impl<'a> SendBody<'a> {
             BodyInner::Reader(v) => v.read(buf),
             BodyInner::OwnedReader(v) => v.read(buf),
             BodyInner::Body(v) => v.read(buf),
+            BodyInner::SizedReader(v, _) => v.read(buf),
```

Comment:
I think we should make both `Reader` and `OwnedReader` have an extra `Option<ContentLength>` instead of introducing a new enum type here.